assets/src/edit-story/utils/getColorPickerActions.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import ColorPickerActions from '../components/panels/colorPreset/colorPickerActions';
+
+function getColorPickerActions(color) {
```

Comment:
This is a bit odd perhaps but wanted to add setting the color presets without color picker itself having to deal with updating the story. Passing optional actions seemed to be reasonable.

Reply:
Alternate options include a context (but with the color picker being in the sidebar, it would have to be quite high in the tree) or render props. Render props would be the more common approach, but this might work too.

Reply:
It kind of is a render prop actually, looking at it with lenient eyes. So you're in the clear 😉